kr/README.md

```diff
@@ -0,0 +1,37 @@
+# Krebsregister
+## Overview
+
+The **Krebsregister** project is designed to connect cancer registries across Germany by providing an overview of federated data. This allows for efficient querying and aggregation of data from multiple sources, empowering cancer research and healthcare analysis at a federal state level.
+
+This project integrates several key modules to enable secure, scalable, and flexible data querying across federated registries. The core components used in this setup are:
+
+- Blaze FHIR Server: For managing FHIR-based medical data.
+- Focus: A tool for executing federated queries.
+- Beam Client: To connect the registry to the federated network.
+- oBDS2FHIR: For automated ETL (Extract, Transform, Load) of oBDS data into FHIR-compatible formats.
+
+## Modules in Use
+- Blaze FHIR Server
+- Focus
+- Beam Client
+- oBDS2FHIR-REST
```

Comment:
A bit redundant with the previous paragraph. Maybe make it useful by providing the links to the repositories?

Reply:
Thx, I messed something up there. Commit is on the way